doc/getting_start.md

```diff
@@ -20,22 +27,29 @@
 ```
 $ git clone git@github.com:SurveyMoonBear/SurveyMoonbear_APP.git
 ```
+   
 
 2. Install gems
 ```
-$ bundle install --without production
+$ bundle config set --local without 'production'
+$ bundle install
+```
+
+> The current project's version of ruby is 3.0.2 , if you don't have the version , you should install it first.
+``` 
+$ rbenv install 3.0.2
 ```
 
 > if it bumps into this problem 
 > `Could not open library 'sodium' cannot load such file -- rbnacl`,
 > please [install libsodium](https://github.com/RubyCrypto/rbnacl/wiki/Installing-libsodium) first
 
-3. Add config/secret.yml file
+3. Add config/secrets.yml file
 
 4. Run DB migrations
 ```
 $ rake db:migrate
-$ RACK_ENV=test db:migrate
+$ RACK_ENV=test rake db:migrate
```

Comment:
I recommend using `RACK_ENV=test bundle exec rake db:migrate` to ensure the right gems are used.